paycalculator/Program.cs

```diff
@@ -1,36 +1,48 @@
 using System;
 using Employees.Entites;
+using Employees;
 using Employees.Services;
 
 namespace paycalculator
 {
     class Program
     {
-        
 
         public static void Main(string[] args)
         {
+            PermanentEmployee permanentEmployee1 = new PermanentEmployee(name: "Joe Bloggs", 40000, 5000, holidayAllowance: 21);
+            PermanentEmployee permanentEmployee2 = new PermanentEmployee(name: "John Smith", 45000, 2500, 21);
+            Console.WriteLine(permanentEmployee1.AllowanceRemaning(4));
+            Console.WriteLine($"The total of money made with bouns is £{permanentEmployee1.CalculateAnnualBounsPay()}");
+            Console.WriteLine($"Hourly pay for a permanet employee is £{permanentEmployee1.HourlyPay()}");
+            TempEmployee tempEmployee1 = new TempEmployee(name: "Clares Jones", annualsalary: 0, annualBonus: 0, dayRate: 350, weeksWorked: 40);
+            Console.WriteLine($"The money made in total is £{tempEmployee1.MoneyMadeInTotal()}");
+            Console.WriteLine($"The hourly pay for temp is £{tempEmployee1.HourlyPay()}");
+            DisplayContent(permanentEmployee1, permanentEmployee2, tempEmployee1);
 
 
-            PermanentEmployee p1 = new PermanentEmployee(name: "Joe Bloggs", 40000, 5000, holidayAllowance: 21);
-            string jsonString = JsonConvert.SerializeObject(p1);
-            Console.WriteLine(jsonString);
+            PermanentEmployeeJSON peo = new PermanentEmployeeJSON();
+            peo.WriteToEmployeeFile(peo.PermanentEmployee6);
+            foreach (var readPermanentFile in peo.ReadFromEmployeeFile())
+            {
+                Console.WriteLine(readPermanentFile);
+            }
 
-            PermanentEmployee p2 = new PermanentEmployee(name:"John Smith", 45000, 2500,21);
-            Console.WriteLine(p1.AllowanceRemaning(4));
-            Console.WriteLine($"The total of money made with bouns is £{p1.CalculateAnnualBounsPay()}");
-            Console.WriteLine($"Hourly pay for a permanet employee is £{p1.HourlyPay()}");
-            TempEmployee t1 = new TempEmployee(name:"Clares Jones", annualsalary:0, annualBonus:0, dayRate:350,weeksWorked: 40);
-            Console.WriteLine($"The money made in total is £{t1.MoneyMadeInTotal()}");
-            Console.WriteLine($"The hourly pay for temp is £{t1.HourlyPay()}");
-            DisplayContent(p1,p2,t1);
-         }
+            TempEmployeeJSON teo = new TempEmployeeJSON();
+            Console.WriteLine("The current directory is:" + teo.path);
+            teo.WriteToEmployeeFile(teo.TempEmployee2);
 
-        public static void DisplayContent(PermanentEmployee p1, PermanentEmployee p2, TempEmployee t1)
+            foreach (var readTempEmployeeFile in teo.ReadFromEmployeeFile())
+            {
+                Console.WriteLine(readTempEmployeeFile);
+            }
+        }
+
+        static void DisplayContent(PermanentEmployee p1, PermanentEmployee p2, TempEmployee t1)
         {
```

Comment:
When refactoring the display, consider looking at use of the formatting instructions for the string.format methods to space fields evenly, rather than using literal spaces.  [See here](https://docs.microsoft.com/en-us/dotnet/standard/base-types/formatting-types#the-tostring-method-and-format-strings)

Reply:
Alright I will do that today thank you.